src/StatusQ/Controls/Validators/StatusRegularExpressionValidator.qml

```diff
@@ -51,10 +51,9 @@ StatusValidator {
 
     name: "regex"
     errorMessage: "Please enter a valid regular expression."
-    validatorObj: RegularExpressionValidator { regularExpression: root.regularExpression }
 
     validate: function (t) {
         // Basic validation management
-        return root.validatorObj.validate() === RegularExpressionValidator.Acceptable
+        return new RegExp(root.regularExpression).test(t)
```

Comment:
Seems like @stefandunca has done a similar fix here https://github.com/status-im/StatusQ/pull/576/files#diff-ffa848f7eb38f5a1379b105eab7afff658a8558240d4b5a0161b85ed0ed36d19R58

Reply:
Nooooo.... :D 
Your PR is scoped smaller, so it's easier to land. Could've updated yours to match those changes and Stefan could've rebased on top of yours

Reply:
lol, ok then, reopening 😆